docker-compose.yml

```diff
@@ -93,6 +93,20 @@ services:
         aliases:
           - devm
 
+  image-manager:
+    image: dojot/image-manager:latest
+    restart: always
+    depends_on:
```

Comment:
mmm, this will depend on postgres and minio only, right?

Reply:
yes, but so far there is no minio

Reply:
done!